modules/health_quest/app/controllers/health_quest/v0/pgd_ques_responses_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  module V0
+    class PgdQuesResponsesController < HealthQuest::V0::BaseController
+      def show
```

Comment:
This is probably just a stylistic difference, but I think we could drop some of these assignments and things are a little clearer like this:
```
      def show
        qresponse = ques_response_by_id
        render json: HealthQuest::V0::PGDQuesResponseSerializer.new(qresponse[:data], meta: qresponse[:meta])
      end
```